src/services/activityReports.js

```diff
@@ -471,6 +471,8 @@ export async function activityReportAlerts(userId, {
         // eslint-disable-next-line quotes
         `(SELECT "Grantees".name as granteeName FROM "Grantees" INNER JOIN "ActivityRecipients" ON "ActivityReport"."id" = "ActivityRecipients"."activityReportId" JOIN "Grants" ON "Grants"."id" = "ActivityRecipients"."grantId" AND "Grantees"."id" = "Grants"."granteeId" order by granteeName ${sortDir} limit 1)`,
       ),
+      // eslint-disable-next-line quotes
+      [sequelize.literal(`(SELECT  CASE WHEN COUNT(1) = 0 THEN '0' ELSE  CONCAT(SUM(CASE WHEN COALESCE("ActivityReportApprovers".status,'needs_action') = 'approved' THEN 1 ELSE 0 END), ' of ', COUNT(1)) END FROM "ActivityReportApprovers" WHERE "ActivityReportApprovers"."activityReportId" = "ActivityReport"."id" AND "deletedAt" IS NULL limit 1)`), 'pendingApprovals'],
```

Comment:
This feels more like a concern of the UI then of the API.

Reply:
This was originally written on the front end but was moved to the backed end. This is because certain approvers don't have the complete list of approvers and status from the alerts sql.